rotkehlchen/icons.py

```diff
@@ -167,10 +176,8 @@ def get_icon(
         If not, all icons of the asset are queried from coingecko and cached
         locally before the requested data are returned.
         """
-        # First search custom icons
-        custom_icon_path = self.custom_iconfile_path(asset)
-        if custom_icon_path is not None:
-            return custom_icon_path
```

Comment:
I removed this since is duplicated logic from `asset_icon_path`

Reply:
Do we have a test for cusom icon paths? Make sure we do if we don't already

Reply:
yes, is in `test_upload_custom_icon`